easybuild/easyconfigs/b/binutils/binutils-2.26-GCCcore-5.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.26'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '5.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked instatically, so not a runtime dep
+    ('zlib', '1.2.8'),
+    # use same binutils version that was used when building GCC toolchain, to 'bootstrap' this binutils
+    ('binutils', version, '', True)
+]
+
+# statically link with zlib, to avoid runtime dependency on zlib
+preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
+prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
```

Comment:
Hah, it's not (to `libz`)...

```
$ ldd /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/binutils/2.26-GCCcore-5.3.0/bin/ar
	linux-vdso.so.1 =>  (0x00007fff1d180000)
	/usr/$LIB/libsnoopy.so => /usr/lib64/libsnoopy.so (0x00002b331abeb000)
	libbfd-2.26.20160125.so => /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/binutils/2.26-GCCcore-5.3.0/lib/libbfd-2.26.20160125.so (0x00002b331a9eb000)
	libz.so.1 => /lib64/libz.so.1 (0x00002b331adfa000)
	libfl.so.2 => /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/flex/2.6.0-GCCcore-5.3.0/lib/libfl.so.2 (0x00002b331b010000)
	libm.so.6 => /lib64/libm.so.6 (0x00002b331b013000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00002b331b297000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00002b331b4b4000)
	libc.so.6 => /lib64/libc.so.6 (0x00002b331b6b9000)
	/lib64/ld-linux-x86-64.so.2 (0x00002b331a9c9000)
```

It is for binutils 2.25 w/ GCCcore 4.9.3 though:

```
 ldd /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/binutils/2.25-GCCcore-4.9.3/bin/ar
	linux-vdso.so.1 =>  (0x00007ffc3afff000)
	/usr/$LIB/libsnoopy.so => /usr/lib64/libsnoopy.so (0x00002b09d3447000)
	libbfd-2.25.so => /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/binutils/2.25-GCCcore-4.9.3/lib/libbfd-2.25.so (0x00002b09d3247000)
	libfl.so.2 => /user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/flex/2.5.39-GCCcore-4.9.3/lib/libfl.so.2 (0x00002b09d337e000)
	libm.so.6 => /lib64/libm.so.6 (0x00002b09d3656000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00002b09d38db000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00002b09d3af8000)
	libc.so.6 => /lib64/libc.so.6 (0x00002b09d3cfc000)
	/lib64/ld-linux-x86-64.so.2 (0x00002b09d3225000)
```

any ideas @wpoely86?